chapter_recurrent-modern/encoder-decoder.md

```diff
@@ -25,6 +25,20 @@ class Encoder(nn.Block):
         raise NotImplementedError
 ```
 
+```{.python .input  n=1}
```

Comment:
Looks like the tabs were not cleared out. Can you restart and clear the outputs of the notebook and then run `d2lbook activate all`.

This should look like -> ```{.python .input}

Reply:
@ypandya please address these as well.

Reply:
I did follow these instructions, there was no change here even after clearing and activating. I checked that there are many such lines in most of the existing notebooks for the mxnet portions. These do not get updated by clearing the output. This is not a part of the output but a part of the input of the cell. Due to this many code cells have some number on them when we open even without any output.

Reply:
Yeah, it is definitely only the cell number so that is cool. But I saw that these change to nil when we restart and clear outputs. Anyways will merge this as Mu has already approved the PR.